MODULE.bazel

```diff
@@ -0,0 +1,5 @@
+module(
+    name = "nlohmann_json",
+    version = "3.11.3",
```

Comment:
Do no include a version here - this has still to be done via BCR patch file - abseil uses version = "head" but this does also no make to much sense since if you go back to an very old version it will still head - I would not include a version here since its always somehow debatable

Reply:
Do you mean this ?
```suggestion
    version = "",
```

Reply:
Or this ?
```suggestion
    version = "0.0.0",
```

Reply:
would delete the whole version line (since it can easily happen that Catch2 version is updated, but one is forgetting about the Bazel build):
```
module(
    name = "nlohmann_json",
    compatibility_level = 1,
)
```
The correct version can than be introduced via a patch file in BCR. But this is opinionated. Different project handle it differently (e.g. rules_go, abeil-cpp, etc.) - was just an idea to leave version out of this file to not get it wrong